ban/tests/http/test_municipality.py

```diff
@@ -29,13 +29,12 @@ def test_get_municipality_with_postcodes(get, url):
     assert resp.status == falcon.HTTP_200
     assert resp.json['id']
     assert resp.json['name'] == 'Cabour'
-    assert resp.json['postcodes'] == [{
```

Comment:
It's better to check the whole object so we are sure there are no unwanted keys inside.